app/javascript/dashboard/i18n/locale/en/auditLogs.json

```
@@ -25,6 +25,13 @@
       "EDIT": "%{agentName} updated an automation rule (#%{id})",
       "DELETE": "%{agentName} deleted an automation rule (#%{id})"
     },
+    "ACCOUNT_USER": {
+      "ADD": "%{agentName} invited (#%{invitee}) to the account as an (#%{role})",
+      "EDIT": {
+        "SELF": "%{agentName} changed their %{attributes} to %{values}",
+        "OTHER": "%{agentName} changed %{attributes} of (#%{user}) to %{values}"
+      }
+    },
```

```suggestion
    "ACCOUNT_USER": {
      "ADD": "%{agentName} invited %{invitee} to the account as an %{role}",
      "EDIT": {
        "SELF": "%{agentName} changed their %{attributes} to %{values}",
        "OTHER": "%{agentName} changed %{attributes} of %{user} to %{values}"
      }
    },
```

Don't think we need to put these in brackets